stable_baselines/logger.py

```diff
@@ -206,7 +208,14 @@ def summary_val(key, value):
             kwargs = {'tag': key, 'simple_value': float(value)}
             return self._tf.Summary.Value(**kwargs)
 
-        summary = self._tf.Summary(value=[summary_val(k, v) for k, v in kvs.items()])
+        def valid_value(value):
```

Comment:
missing dosctring and I'm not sure if this function should be defined only in this local context

Reply:
I agree, I would typically put a function like this at the top level rather than inside the method, but the existing code already had a very similar kind of function defined here, so I copied the style. I can add a doc string. Let me know if I should move both functions out of this method.

Reply:
Yes, please move the two functions.

>but the existing code already had a very similar kind of function defined here

legacy code from OpenAI ....